doc/COMPILING/COMPILING-CYGWIN.md

```diff
@@ -1,71 +1,89 @@
 
-3. After CYGWIN installation is complete select following packages and press `Next` button to download and install them:
+3. Give the `\downloads\` folder as the local package directory (e.g. `C:\dev\cygwin64\downloads`).
 
-* `wget`.
+4. Use system proxy settings unless you know you have a reason not to.
 
-4. System will tell you it will install following packages (you will install everything else later):
+5. On the next screen, select any mirror you like.
 
-```
-...
-```
+6. Enter `wget` into the search box, expand the "Web" category and select the latest version in the drop-down (1.21.1-1 as of the time this guide was written).
 
-5. Check boxes to add shortcuts to Start Menu and/or Desktop, then press `Finish` button.
+7. Confirm that wget is shown in the following screen by scrolling to the bottom. This is the full list of packages that Cygwin will download and install.
+
+8. Retry any packages that throw errors.
+
+9. Check boxes to add shortcuts to Start Menu and/or Desktop, then press `Finish` button.
 
 ## Configuration:
 
-1. Install `apt-cyg` for easier package installation with:
+1. Launch the Cygwin64 terminal from your desktop.
+
+2. Install `apt-cyg` for easier package installation:
 
 ```bash
 wget rawgit.com/transcode-open/apt-cyg/master/apt-cyg -O /bin/apt-cyg
 chmod 755 /bin/apt-cyg
 ```
 
-2. Install packages required for compilation with:
+3. Install packages required for compilation:
 
 ```bash
-apt-cyg install git make astyle gcc-g++ libintl-devel libiconv-devel libSDL2_image-devel libSDL2_ttf-devel libSDL2_mixer-devel libncurses-devel xinit
+apt-cyg install astyle ccache gcc-g++ gettext-devel git libiconv-devel libintl-devel libSDL2_image-devel libSDL2_mixer-devel libSDL2_ttf-devel make xinit
 ```
 
+You will see messages saying packages are already installed, as well as Cygwin installing packages you didn't ask for; this is the result of Cygwin's package manager automatically resolving dependencies.
+
 ## Cloning and compilation:
 
-1. Clone Cataclysm-DDA repository with following command line:
+1. Clone the Cataclysm-DDA repository with following command:
 
-**Note:** This will download whole CDDA repository. If you're just testing you should probably add `--depth=1`.
+**Note:** This will download the entire CDDA repository and all of its history (3GB). If you're just testing, you should probably add `--depth=1` (~350MB).
 
 ```bash
+cd /cygdrive/c/dev
 git clone https://github.com/CleverRaven/Cataclysm-DDA.git
 cd Cataclysm-DDA
 ```
 
-2. Compile with following command line:
+2. Compile:
 
 ```bash
-make CCACHE=1 RELEASE=1 CYGWIN=1 DYNAMIC_LINKING=1 SDL=1 TILES=1 SOUND=1 LOCALIZE=1 LANGUAGES=all LINTJSON=0 ASTYLE=0 RUNTESTS=0
+make CCACHE=1 RELEASE=1 Cygwin=1 DYNAMIC_LINKING=1 SDL=1 TILES=1 SOUND=1 LOCALIZE=1 LANGUAGES=all LINTJSON=0 ASTYLE=0 BACKTRACE=0 RUNTESTS=0
```

Comment:
I see no value in changing macro name.

Reply:
Whoops. I global-replaced "CYGWIN" with "Cygwin" in other places, and forgot to change this one back.